next/app/about/leadership/page.tsx

```diff
@@ -0,0 +1,194 @@
+// import MemberCard from './MemberCard/page';
+import { GitHub, LinkedIn, Email } from "../../../components/common/Icons";
+export default function Leadership() {
+  const team = {
```

Comment:
I like that you have the 'content' pulled out into a data structure. Let's clean up this file some more by moving this array into a separate file in the same folder (a sibling to page.tsx)

Also, could you utilize typescript for each entry? You could create an 'officer.ts' file in the types folder, which defines all the fields. Then, you would use type annotations in your list declaration.